packages/css/src/card/card.css

```diff
@@ -31,12 +32,14 @@
     width: fit-content;
 
     .hds-card__media__img {
+      border-radius: var(--hds-border-radius-8);
```

Comment:
I think this should be moved up to `hds-card__media` with `overflow: hidden`. This way it will work for other types of content

Reply:
Done